File: Easy/two_number_sum.py
def twoNumberSum(array, targetSum):
	output = []
    for i in enumerate(array):
		for j in enumerate(array):
			if i[0] != j[0]:
				if i[1] + j[1] == targetSum:
					return [i[1], j[1]]
	return output

File: Easy/Two_Number_Sum/two_number_sum.java
import java.util.*;

class Program {
  public static int[] twoNumberSum(int[] array, int targetSum) {
	//#Loop through the main array
    for(int i = 0; i < array.length; i++){
			//#Second loop to step to the right direction
			for(int j = 0; j < array.length; j++){
				//#make sure that we're not adding the same index twice
				if(i != j){
					if(array[i] + array[j] == targetSum){
						return new int[] {array[i], array[j]};
					}
				}
			}
		}
    return new int[0];
  }
}
